.github/workflows/triage_issues.yml

```
@@ -1,4 +1,4 @@
-name: "Issue triage"
+name: 'Triage: Issues'

```

Perhaps 
```suggestion
name: 'Issues: Auto-triage'
```

Because it is more likely that we'll add more workflows about issues than about triaging things

(Oops, just too late... maybe for next PR)

---

I went with the "Triage" prefix because I figured we might triage PRs at some point, but we can always change to the "Issue" prefix if we end up with lots of issue-based workflows.